### PR TITLE
Onboarding: Review information screen

### PR DESCRIPTION
**Pull request by fabriziovigevani**
closes https://github.com/1Hive/gardens/issues/151

### CODE REVIEW

File: src/components/Onboarding/Screens/ReviewInformation.js [hunk 1]
```diff
@@ -19,4 +63,544 @@ function ReviewInformation({ title }) {
...
+      css={`
+        padding: ${5 * GU}px ${7 * GU}px;
+        width: 100%;
+      `}
+    >
+      <ReviewTokens />
+      <LineBreak />
+      <ReviewHoneyswapLiquidity />
+      <LineBreak />
+      <ReviewIssuance />
```
Comment (0xGabi) [inline src/components/Onboarding/Screens/ReviewInformation.js:212]: We need to check if we are on a Pre-existing token Garden then do not show this.

File: src/components/Onboarding/Screens/ReviewInformation.js [hunk 2]
```diff
@@ -19,4 +63,544 @@ function ReviewInformation({ title }) {
...
+            label="Token address"
+            value={
+              <div>
+                {config.tokens.address}{' '}
+                <span>({config.tokens.existingTokenSymbol})</span>
+              </div>
+            }
+          />
+        )}
+        <Field label="Token name" value={config.tokens.name} />
```
Comment (0xGabi) [inline src/components/Onboarding/Screens/ReviewInformation.js:253]: I notice that the Token address and name are super close without padding. What do you think of moving the Token address on the first row? And then the Token name and symbol on a second row.

Comment (fabriziovigevani) [reply src/components/Onboarding/Screens/ReviewInformation.js:253]: Do you think it could help to have the token address in an `IdentityBadge` ?

**Comment by fabriziovigevani** [comment]
Pushed the review changes

There a couple of things yet to do regarding styling the Accordion component but will need to be updated at UI kit level: 

- Modify Accordion item content background
- Make the left border indicator when an item is opened, to follow the container radius for first and last items

File: src/components/Onboarding/Screens/ReviewInformation.js (hunk 2 above)
Comment (0xGabi) [reply src/components/Onboarding/Screens/ReviewInformation.js:264]: I don't think is super relevant. Only to make it easy for the user to click the block explorer and check is the right one.

**Review by 0xGabi — APPROVED** [review]

**Pull request merged**